src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/Tool/AppProvisioningTool.cs

```diff
@@ -46,34 +41,32 @@ public class AppProvisioningTool : IMsAADTool
         private ProjectDescriptionReader? _projectDescriptionReader;
         private ProjectDescriptionReader ProjectDescriptionReader => _projectDescriptionReader ??= new ProjectDescriptionReader(FilePaths);
 
-        public AppProvisioningTool(string commandName, ProvisioningToolOptions provisioningToolOptions)
+        public AppProvisioningTool(string commandName, ProvisioningToolOptions provisioningToolOptions, bool silent = false) // TODO silent is temporary
```

Comment:
Yeah, for now this is because VS is expecting a single JSONResponse, but the blazor wasm hosted scenario does some extra behind the scenes work. The plan is to refactor so that we can output all of the updates and logging to the console and VS can log in the summary page as a stream instead of a blob, opened an issue: https://github.com/dotnet/Scaffolding/issues/1879